src/bin/enqueue-job.rs

```diff
@@ -5,7 +5,11 @@ use diesel::PgConnection;
 fn main() -> AppResult<()> {
     let conn = db::connect_now()?;
     let mut args = std::env::args().skip(1);
-    match &*args.next().unwrap_or_default() {
+
+    let job = args.next().unwrap_or_default();
+    println!("Enqueueing background job: {}", job);
```

Comment:
When called locally without a command-line argument, the eror output isn't great, but I guess it's good enough – this isn't a tool meant to be used by many end users.